examples/two-columns-layout/src/components/Icons.tsx

```diff
@@ -0,0 +1,56 @@
+/** @jsx h */
+import { h } from 'preact';
+
+type IconProps = {
+  className?: string;
+};
+
+export const ChevronRightIcon = ({ className }: IconProps) => (
```

Comment:
Applicable to the other icons.

```suggestion
export const ChevronRightIcon = (props: React.SVGProps<SVGSVGElement>) => (
```

Reply:
I went for `props: h.JSX.SVGAttributes<SVGSVGElement>`